versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "<= 4.74, != 4.75.0"
+      version = "~> 6.34"
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = "<= 4.74, != 4.75.0"
```

Comment:
See [Terraform docs about `~>` and version constraints](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#-3).
I'm just preventing bumps to the major number.

We can safely remove `!= 4.75.0` because `~> 6...` ensures that we only use major version 6.

It's not clear why we had `<= 4.74, != 4.75.0` in the first place. See [pull-request where we introduced these constraints](https://github.com/GoogleCloudPlatform/terraform-google-three-tier-web-app/pull/62). But, as long as our integration test passes in this pull-request, we should be good.